app/services/landingPageTourService.ts

```diff
@@ -0,0 +1,120 @@
+import * as angular from 'angular';
+import * as jQuery from 'jquery';
+
+interface ILandingPageTourService {
+  startTour(tourConfig: any);
+}
+
+export class LandingPageTourService implements ILandingPageTourService {
+  public static $inject = ['$document', '$timeout', 'Constants', 'GuidedTourService'];
+
+  private $document: any;
+  private $timeout: any;
+  private Constants: any;
+  private GuidedTourService: any;
+  private tourConfig: any;
+
+  constructor ($document: any, $timeout: any, Constants: any, GuidedTourService: any) {
+    this.$document = $document;
+    this.$timeout = $timeout;
+    this.Constants = Constants;
+    this.GuidedTourService = GuidedTourService;
+
+    this.tourConfig = {
+      id: "getting-started-tour",
+      steps: []
+    };
+
+    let stepConstants = Constants.LANDING_PAGE_TOUR_STEPS || {};
```

Comment:
I don't think it's unreasonable asking people to write some code if they want to do anything advanced like clicking tabs, but yeah custom tours can break if we change the page structure or class names. So I'm on the fence.

@jwforres Thoughts?

I have a feeling admins who customize these will want to be able to completely replace the tours.